packages/components/src/components/primitives/Icons/types.ts

```diff
@@ -1,4 +1,5 @@
 export interface IconProps {
   color?: string;
   boxSize?: string;
+  size?: 's' | 'm' | 'l';
```

Comment:
Could we be consistent and try to adapt this to follow the implementation of the `Button` for example? Instead of `interfaces` use `consts` etc... Check `Button` types for references 👍🏻 Could we reflect this new property in the `story` file?